packages/fast-components-react-msft/src/text-action/text-action.props.ts

```diff
@@ -24,11 +24,7 @@ export interface TextActionHandledProps
     /**
      * The text action button
      */
-    button?: (
-        classname?: string,
-        disabled?: boolean,
-        appearance?: ButtonAppearance
-    ) => React.ReactNode;
+    button?: (classname?: string) => React.ReactNode;
```

Comment:
so do you not want a disabled state set on the root level to be passed here to also disable the button?

Reply:
I accidentally removed this - back in now and working as expected.